addon/components/x-select.js

```diff
@@ -77,7 +77,24 @@ export default Ember.Component.extend({
       this._updateValueSingle();
     }
 
-    this.sendAction('action', this.get('value'), this);
+    this.sendAction('action', this.get('value'), this); // should we deprecate `action` in favor of `on-change`?
+    this.sendAction('on-change', this.get('value'), this);
+  },
+
+  /**
+   * When the click DOM event fires on the element, trigger the
+   * component's action with the jQuery event, x-select value, and the component.
+   */
+  click(event) {
+    this.sendAction('on-click', event, this.get('value'), this);
```

Comment:
Shouldn't the value of the componnent be the first argument? I believe this is the behavior in the components that ship with ember. We should verify what happens when you do both:

```hbs
{{input on-click=(action "doStuff")}}

and

<div on-click=(action "do stuff")>Hmmm....</div>
```